City-of-Helsinki/helsinki-paatokset
# PP-0: Module and package updates.

**To test:**
* Checkout branch
* Run `make new` OR `make composer-install; make drush-updb; make drush-cim;`
* Go to https://helsinki-paatokset.docker.sh/fi/node/add/
  * The "Unit" content type should be missing since it was removed from base config
* Go to https://helsinki-paatokset.docker.sh/fi/admin/reports/updates
  * There should be no pending module updates
